code/go/0chain.net/smartcontract/storagesc/readpool.go

```diff
@@ -140,14 +140,6 @@ type readPoolRedeem struct {
 	Balance state.Balance `json:"balance"` // balance reduction
 }
 
-func toJson(val interface{}) string {
```

Comment:
Why was toJson method removed?

Reply:
it's a minor utility function (not a  struct method to be precise), that was used poorly. Particularly, it was called from the "sub" packages that are candidates for removal (read/writepool). Also it has no "intrinsic" value, from my point of view.